web/static/css/app.css

```diff
@@ -119,6 +119,10 @@ a {
   display: none;
 }
 
+header {
```

Comment:
That's a good link to study about CSS specificity and why all these weird rules with BEM and "no element styles": https://css-tricks.com/strategies-keeping-css-specificity-low/